docs/test-run.md

```diff
@@ -0,0 +1,114 @@
+<!-- markdownlint-disable code-block-style line-length no-bare-urls no-emphasis-as-heading -->
+# Run the Test Suite
+
+The Test Suite can be run using the Certsuite tool directly or through a container.
+
+To run the Test Suite direct use:
+
+```shell
+./certsuite run -l <label-filter> -c <tnf-config> -k <kubeconfig> -o <output-dir> [<flags>]
+```
+
+If the _kubeconfig_ is not provided the value of the `KUBECONFIG` environment variable will be taken by default.
+
+The CLI output will show the following information:
+
+* Details of the Certsuite and claim file versions, the test case filter used and the location of the output files.
+* The results for each test case grouped into test suites (the most recent log line is shown live as each test executes).
+* Table with the number of test cases that have passed/failed or been skipped per test suite.
+* The log lines produced by each test case that has failed.
+
+Once the test run has completed, the test results can be visualized by opening the `results.html` website in a web browser and loading the `claim.json` file.
+
+For more information on how to analyze the results see [Test Output](test-output.md).
+
+## Building the Certsuite tool executable
+
+The Certsuite binary can be built as follows:
+
+```shell
+make build-certsuite-tool
+```
+
+## Test labels
+
+The test cases cases have several labels to allow for different types of groupings when selecting which to run. These are the following:
```

Comment:
Should we also include `all` here as an option?

Reply:
It's explained below:
`To select all the test cases the "all" label filter can be used.`

I think `all` is more of a label filter (that gets expanded in the code into the different categories) than a proper label.